qa-tests-backend/src/main/groovy/objects/ImageIntegration.groovy

```diff
@@ -253,6 +253,36 @@ class QuayImageIntegration implements ImageIntegration {
     }
 }
 
+class GHCRImageIntegration implements ImageIntegration {
+
+    static String name() { "GitHub Container Registry" }
+
+    static Boolean isTestable() {
+        return true
+    }
+
+    static ImageIntegrationOuterClass.ImageIntegration.Builder getCustomBuilder(Map customArgs = [:]) {
+        Map defaultArgs = [
+                name: "ghcr",
+                username: Env.mustGet("GHCR_REGISTRY_USER"),
+                password: Env.mustGet("GHCR_REGISTRY_PASSWORD"),
+        ]
```

Comment:
I assume this would fail if `GHCR_REGISTRY_USER` or `GHCR_REGISTRY_PASSWORD` is not set and I do not see either being set in the various workflows.

I wasn't able to find any e2e runs which executed the associated test(s). From what I can tell all the tests tagged with `BAT` are not being run (including the new ones). Not sure if this is intentional or not.  Looks like the decision for whether or not to run the BAT tests is here: https://github.com/stackrox/stackrox/blob/aad5a8ba63b2844cf7e42a7048c0cbe1615b39ed/qa-tests-backend/scripts/run-part-1.sh#L96-L107

It'd be great to see the new test run / succeed.

Reply:
Yeah I forgot to set the env variables in the CI run and didn't notice because it's not a BAT test, thanks for pointing it out.

Reply:
qa tests should run and pass now.